templates/base.html.twig

```diff
@@ -4,6 +4,8 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
         <link rel="icon" type="image/png" href="{{ asset('build/images/favicon.png') }}"/>
+        <link rel="stylesheet" href="{{ asset('css/app.css') }}">
+        <link href="https://cdnjs.cloudflare.com/ajax/libs/flowbite/1.6.5/flowbite.min.css" rel="stylesheet" />
```

Comment:
No you should'nt had flowbite by the cdn. Install it with yarn

Reply:
modification done